firmware/usbdrv/usbdrv.c

```diff
@@ -601,56 +601,56 @@ uchar           isReset = !notResetState;
 }
 
 /* ------------------------------------------------------------------------- */
-
```

Comment:
Why is usbdrv.c changed? This should not be necessary.

Reply:
Only trailing spaces are affected and usbPoll(void) is commented out, otherwise I got a nasty warning.